### DISCUSSION TITLE
Environments descirptions

### DISCUSSION THREAD

**Discussion by modanesh** [bulletphysics/bullet3#3742] [General]
Where can I find each environment description? Like the observations and actions range and what each value refers to, maximum reward, and considered solved reward etc.

**Comment by Steven89Liu** [comment]
i think you should can get these values from the each env definition from the bullet3/examples/pybullet/gym/pybullet_envs/, but i also didn't find any more detail description.
hope this can help you.

**Comment by modanesh** [comment]
I knew about that. There are also some information here: ```examples/pybullet/gym/pybullet_envs/__init__.py```. But I was looking for something more in detail, like [this](https://github.com/openai/gym/blob/master/gym/envs/classic_control/cartpole.py#L28) for CartPole from OpenAI gym.
